### PR TITLE
Add .editorconfig to workflow-sync.yml.

### PR DESCRIPTION
**Pull request by pondermatic**
## Description
Added syncing of `.editorconfig` to PHP and Utility repositories.

This `.editorconfig` file came straight from LifterLMS core.

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [ ] My code has been tested.
- [ ] My code passes all existing automated tests. 
- [ ] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

File: .github/workflow-templates/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://developer.wordpress.org/coding-standards/wordpress-coding-standards/
```
Comment (thomasplevy) [inline .github/workflow-templates/.editorconfig:17]: Please add a comment like this so it's clear that the file is managed from outside the repos: https://github.com/gocodebox/.github/blob/a38fa0c94402895bd12af96b6cfb7dfb27a4c08e/.github/workflow-templates/coding-standards.yml#L1-L11

**Review by thomasplevy — CHANGES_REQUESTED** [review]

**Comment by pondermatic** [comment]
I wasn't sure what the best place to put `.editorconfig` was. Is `workflow-templates` OK? It's a template, and it is copied by a workflow, but doesn't do any workflow in the destination repo.

File: .github/workflows/workflow-sync.yml [hunk 2]
```diff
@@ -47,6 +47,7 @@ env:
     gocodebox/lifterlms-stripe
 
   PHP_FILES: |
+    .github/workflow-templates/.editorconfig=.editorconfig
```
Comment (thomasplevy) [inline .github/workflows/workflow-sync.yml]: what repo am I missing that requires it to be duplicated in both util and php file lists?

Comment (pondermatic) [reply .github/workflows/workflow-sync.yml]: These are the ones I noticed. This weekend I looked into making a comprehensive list and refactoring the env.repo lists, but thought maybe that should be a project for later.

gocodebox/lifterlms-cli
gocodebox/lifterlms-labs
gocodebox/lifterlms-launchpad
gocodebox/lifterlms-launchpad-child
gocodebox/lifterlms-tests
gocodebox/lifterlms-zapier
gocodebox/llms-academy-rest
gocodebox/sky-pilot
gocodebox/snippets
gocodebox/lifterlms-com-experiments
gocodebox/lifterlms-com-reporting
gocodebox/lifterlms.com-rest-api
gocodebox/lifterlms-com-support-desk

snippets is the one that got me started on this.

Comment (thomasplevy) [reply .github/workflows/workflow-sync.yml]: I just did a diff on the two lists and there's no projects in the PHP list that aren't also listed in the UTIL list... So I'm suggesting that the file isn't needed in the PHP list (as it'll be picked up by the UTIL list)

Agree?

Comment (pondermatic) [reply .github/workflows/workflow-sync.yml]: Ah! Yes, I agree.

**Pull request merged**